components/builder-originsrv/src/migrations/origin_public_keys.rs

```diff
@@ -83,11 +83,15 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         RETURN;
                     END
                     $$ LANGUAGE plpgsql STABLE"#)?;
-
-    migrator.migrate("originsrv",
-                     r#"ALTER TABLE origin_public_keys
+    migrator
+        .migrate("originsrv",
+                 r#"ALTER TABLE origin_public_keys
                         DROP CONSTRAINT IF EXISTS
                           origin_public_keys_full_name_key"#)?;
-
+    migrator
```

Comment:
I agree with @mwrock... a dedupe statement in the migration should be fine. It should be possible to write one that's idempotent.

It's certainly easier to not have to remember to apply the fix manually in each environment.